content/references/rippled-api/admin-rippled-methods/logging-and-data-management-methods/crawl_shards.md

```diff
@@ -0,0 +1,134 @@
+
+_The `crawl_shards` method is an [admin method](admin-rippled-methods.html) that cannot be run by unprivileged users._
+
+### Request Format
+
+An example of the request format:
+
+<!-- MULTICODE_BLOCK_START -->
+
+*WebSocket*
+
+```json
+{
+  "command": "crawl_shards",
+  "pubkey": true,
+  "limit": 2
+}
+```
+
+*JSON-RPC*
+
+```json
+{
+  "method": "crawl_shards",
+  "params": [
+    {
+      "pubkey": true,
+      "limit": 2
+    }
+  ]
+}
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+
+The request includes the following fields:
+
+| `Field`  | Type    | Description                                             |
+|:---------|:--------|:--------------------------------------------------------|
+| `pubkey` | Boolean | _(Optional)_ If `true`, the response includes the node public keys (for peer-to-peer communications) of servers that were crawled. The default is `false`. |
+| `limit`  | Number  | _(Optional)_ How many hops deep to search. The default is 0, which searches direct peers only. With a limit of `1`, searches peers' peers also. (The number of peers searched potentially grows exponentially as `limit` increases.) The maximum value is `3`. |
+
+
+### Response Format
+
+An example of a successful response:
+
+<!-- MULTICODE_BLOCK_START -->
+
+*WebSocket*
+
+```json
+{
+  "result": {
+    "complete_shards": "1-2,5,8-9,584,1973,2358",
+    "peers": [
+      {
+        "complete_shards": "1-2,8,47,371,464,554,653,857,1076,1402,1555,1708,1813,1867",
+        "public_key": "n9LxFZiySnfDSvfh23N94UxsFkCjWyrchTeKHcYE6tJJQL5iejb2"
+      },
+      {
+        "complete_shards": "8-9,584",
+        "ip": "192.168.1.132",
+        "public_key": "n9MN5xwYqbrj64rtfZAXQy7Y3sNxXZJeLt7Lj61a9DYEZ4SE2tQQ"
+      }
+    ]
+  },
+  "status": "success",
+  "type": "response"
+}
+```
+
+
+*JSON-RPC*
+
+```json
+200 OK
+
+{
+  "result": {
+    "complete_shards": "1-2,5,8-9,584,1973,2358",
+    "peers": [
+      {
+        "complete_shards": "1-2,8,47,371,464,554,653,857,1076,1402,1555,1708,1813,1867",
+        "public_key": "n9LxFZiySnfDSvfh23N94UxsFkCjWyrchTeKHcYE6tJJQL5iejb2"
+      },
+      {
+        "complete_shards": "8-9,584",
+        "ip": "192.168.1.132",
+        "public_key": "n9MN5xwYqbrj64rtfZAXQy7Y3sNxXZJeLt7Lj61a9DYEZ4SE2tQQ"
+      }
+    ],
+    "status": "success"
+  }
+}
+```
+
+
+<!-- MULTICODE_BLOCK_END -->
+
+The response follows the [standard format][], with a successful result containing the following fields:
+
+| `Field`           | Type   | Description                                     |
+|:------------------|:-------|:------------------------------------------------|
+| `complete_shards` | String | The range of history shards that are available across all servers that were searched. This may be disjointed. For example, `1-2,5,7-9` indicates that shards 1, 2, 5, 7, 8, and 9 are available. |
```

Comment:
This fields describes the shards the server issued the command to may hold.

A possible description could be...
``` The range of history shards the server issued the command has available. This may be disjointed. For example, `1-2,5,7-9` ```